modules/nf-core/kraken2/add/main.nf

```diff
@@ -0,0 +1,38 @@
+process KRAKEN2_ADD {
```

Comment:
~Is there a reason why this is called `KRAKEN2_ADD` rather than `KRAKEN2_BUILD`? I would argue for the latter - the only difference between add/build etc is the `--add-to-library` which can be supplied either via a prefix, or as it's an essential 'parameter' can be specified as an input channel (e.g. called `mode`?).~

Reply:
Although looking at the parametmer list maybe this inded makes more sense as after `--build` I would immediately run the `--clean`  flag in same module...

OK so this is fine :sweat_smile: ignore the comment above (leaving for food for thought anyway)

Reply:
Or you could do the `mode` input channel method above, and wrap the `--clean` in a bash `if` statement... but I guess then someone couldn't then use the module for taxonomy downloading (I think?... it's 6:30am here... sorry :sweat_smile: )...